pacman/operations/routing_table_generators/merged_routing_table_generator.py

```diff
@@ -151,16 +136,14 @@ def __init__(self, iterable):
             self.__next = None
             self.__has_next = False
 
-    @property
     def peek(self):
         return self.__next
 
     @property
     def has_next(self):
         return self.__has_next
 
-    @property
-    def next(self):
+    def pop(self):
```

Comment:
I approve of making non-idempotent operations not be properties. 👍

Reply:
Was needed as otherwise using a debugger remove a value from the list!